diff --git a/node_modules/react-native-pie/src/Pie.js b/node_modules/react-native-pie/src/Pie.js
index 8689447..619ed79 100644
--- a/node_modules/react-native-pie/src/Pie.js
+++ b/node_modules/react-native-pie/src/Pie.js
@@ -54,15 +54,15 @@ const ArcShape = ({dimensions, color, strokeCap, startAngle, arcAngle, isBezian}
     innerRadius,
   );
   const strokeWidth = isBezian ? (arcAngle * 5) : width;
-  return <Shape d={path} stroke={color} strokeWidth={strokeWidth} strokeCap={strokeCap} />;
+  return <Shape d={path} stroke={color} strokeWidth={strokeWidth} strokeCap={strokeCapForLargeBands} />;
 };
 
 //The initial band to set the backgroundColor behind the pie chart
 const Background = ({dimensions, color}) => {
-  return <ArcShape dimensions={dimensions} color={color} startAngle={0} arcAngle={360} />
+  return <ArcShape dimensions={dimensions} color={color} startAngle={0} arcAngle={300} />
 }
 
-const getArcAngle = (percentage) => percentage / 100 * 360;
+const getArcAngle = (percentage) => percentage / 100 * 300;
 const shouldShowDivider = (sections, dividerSize) => sections.length > 1 && !Number.isNaN(dividerSize);
 
 const Sections = ({dimensions, paintedSections, sections, shouldShowRoundDividers, strokeCapForLargeBands}) => {
@@ -166,7 +166,7 @@ const Pie = ({ sections, radius, innerRadius, backgroundColor, strokeCap, divide
   
   return (
     <Surface width={radius * 2} height={radius * 2}>
-      <Group rotation={-90} originX={radius} originY={radius}>
+      <Group rotation={117} originX={radius} originY={radius}>
         <Background dimensions={dimensions} color={backgroundColor} />
         <Sections 
           dimensions={dimensions} 
